tests/labhub_test.py

```diff
@@ -48,23 +48,17 @@ def test_invite_cmd(self):
 
         self.mock_team.is_member.return_value = True
         plugins.labhub.os.environ['GH_TOKEN'] = 'patched?'
-        testbot.assertCommand('!invite meet to developers',
-                                   '@meet, you are a part of developers')
+        testbot.assertCommand('!invite meet to developers', '@meet, you are a part of developers')
         self.assertEqual(labhub.TEAMS, teams)
-        testbot.assertCommand('!invite meet to something',
-                                   'select from one of the')
+        testbot.assertCommand('!invite meet to something', 'select from one of the')
 
         self.mock_team.is_member.return_value = False
 
-        testbot.assertCommand('!invite meet to developers',
-                                   ':poop:')
+        testbot.assertCommand('!invite meet to developers', ':poop:')
 
-        testbot.assertCommand('!invite meetto newcomers',
-                                   'Command "invite" / "invite meetto" not found.')
+        testbot.assertCommand('!invite meetto newcomers', 'Command "invite" / "invite meetto" not found.')
```

Comment:
Line length 80 chars and just correct the indentation. Eg:
```
 testbot.assertCommand('!invite meetto newcomers',
                       'Command "invite" / "invite meetto" not found.')
```

Reply:
Ok!